dbt/adapters/snowflake/impl.py

```diff
@@ -363,6 +363,8 @@ def submit_python_job(self, parsed_model: dict, compiled_code: str):
         if use_anonymous_sproc:
             proc_name = f"{identifier}__dbt_sp"
             python_stored_procedure = f"""
+ALTER SESSION SET LOG_LEVEL = DEBUG;
```

Comment:
@manishkundu @wenbaolicat I think if you do this at the account level, you'll get the behavior you want. I know that's a high bar since you need account level access. But that should unblock you until we implement this at the "out-of-the-box" level.

Reply:
Updated: Try running `ALTER ACCOUNT SET TRACE_LEVEL = ON_EVENT` before running your repro, and swap out the sql setting of `LOG_LEVEL` with the `logging` setting (`logger.setLevel(logging.INFO)`). It looks like this is working from my end, which means we may only need to implement the `TRACE_LEVEL` piece.

Reply:
Update 2, it looks like this was a timing issue and this does not work.